molecule/default/tests/test_default.py

```diff
@@ -12,7 +12,23 @@
 ).get_hosts("all")
 
 
-@pytest.mark.parametrize("x", [True])
-def test_packages(host, x):
-    """Run a dummy test, just to show what one would look like."""
-    assert x
+@pytest.mark.parametrize("directory", [{"path": "/var/cyhy/admiral", "mode": "0o755"}])
+def test_packages(host, directory):
+    """Assert that the correct directory was created."""
+    assert host.file(directory["path"]).exists
+    assert host.file(directory["path"]).is_directory
+    assert oct(host.file(directory["path"]).mode) == directory["mode"]
+
+
+@pytest.mark.parametrize(
+    "file",
+    [
+        {"path": "/var/cyhy/admiral/docker-compose.yml", "mode": "0o644"},
+        {"path": "/var/cyhy/admiral/docker-compose-dev.yml", "mode": "0o644"},
```

Comment:
If the `docker-compose-dev.yml` file is removed this parameter must also be removed.

Reply:
Removed in [df4546d](https://github.com/cisagov/ansible-role-admiral/pull/3/commits/df4546d74d5cff17ebea1a154f8b574f5a6c03f7).